typescript/src/entities/products.test.ts

```diff
@@ -10,3 +11,37 @@ test('can get and set id', () => {
   product.id = 2;
   expect(product.id).toBe(2);
 });
+
+test('can get and set name', () => {
+  const product = new Product();
+  product.name = 'example';
+  expect(product.name).toBe('example');
+});
+
+test('can fetch product from server', () => {
+  const testName = 'S7qHUfV_dr5l';
+  mockFetch(true, {'product': {'name': testName}}, 200);
+  (window as any).merchiBackendUri = 'override.example.com';
+  return Product.get(1).then(product => expect(product.name).toBe(testName));
+});
+
+test('handle nonsense from server', () => {
+  // non existent property just ignored. no crash, no update
```

Comment:
Not important thing here but I personally prefer to enhance the test description over add comment.

eg 
> test('non existent property from server just ignored. no crash, no update'